### PR TITLE
Added Thank you for contributing in Malayalam 

### PR DESCRIPTION
**Pull request by NishithP2004**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- [Preview Link]https://nishithp2004.github.io/gci18.fossasia.org/

## Description / Changes
Hi I have successfully added Thank you for contributing in Malayalam.

### CODE REVIEW

File: js/thankyou.js [hunk 1]
```diff
@@ -26,8 +26,9 @@
 'Grazie per aver contribuito!',
 'Go raibh maith agat as do chuid ranníocaíochtaí!',
 'Terimakasih atas kontribusi anda!',
-'Köszönjük a hozzájárulást!'
-];
+'Köszönjük a hozzájárulást!',
+'സംഭാവന ചെയ്തതിന് നന്ദി'
+ ];
```
Comment (nikit19) [inline js/thankyou.js]: indentation. remove extra space

**Comment by NishithP2004** [comment]
I have resolved it ,could you please review it.

**Comment by NishithP2004** [comment]
I am not able to Create a page.

File: CNAME [hunk 2]
```diff
@@ -1 +1 @@
-gci18.fossasia.org
+gci18.fossasia.ni
```
Comment (nikit19) [inline CNAME]: why?

Comment (samagragupta) [inline CNAME]: Please add EOF.

File: js/thankyou.js [hunk 3]
```diff
@@ -27,6 +27,7 @@
 'Go raibh maith agat as do chuid ranníocaíochtaí!',
 'Terimakasih atas kontribusi anda!',
 'Köszönjük a hozzájárulást!',
+'സംഭാവന ചെയ്തതിന് നന്ദി',
```
Comment (abishekvashok) [inline js/thankyou.js]: Wrong spelling :P

File: CNAME (hunk 2 above)
Comment (abishekvashok) [reply CNAME]: Please don't change it

File: js/thankyou.js (hunk 3 above)
Comment (NishithP2004) [reply js/thankyou.js]: Do u mean I Have written it wrong in Malayalam

**Comment by NishithP2004** [comment]
Please check it , I have fixed the PR template.

**Comment by Sid911** [comment]
Please squash your commits... Rest looks good in the template

File: js/thankyou.js (hunk 3 above)
Comment (abishekvashok) [reply js/thankyou.js]: Yes

**Comment by abishekvashok** [comment]
The Malayalam translation is wrong. Please don't merge it, till it's fixed.

**Comment by NishithP2004** [comment]
Could anyone please merge this.

**Comment by abishekvashok** [comment]
It should have been `സംഭാവന ചെയ്തത്തിന്`

**Comment by jogendra** [comment]
@abishekvashok Sorry I missed your comment, should I revert this?

**Comment by abishekvashok** [comment]
Huh, maybe not, we could have added a new pr for it. Anyway since you revrted it; @NishithP2004 can you manage another pr pls?

**Comment by jogendra** [comment]
@NishithP2004 Make another PR with correct malayalam **സംഭാവന ചെയ്തത്തിന്**

**Comment by abishekvashok** [comment]
Make another pr.

**Comment by NishithP2004** [comment]
My PR link : 
https://github.com/fossasia/gci18.fossasia.org/pull/482

**Comment by NishithP2004** [comment]
@abishekvashok @jogendra I have done as requested.

**Pull request merged**